README.md

```diff
@@ -33,8 +34,60 @@ aimed at react developers that want to embed the widgets into their applications
 npm install --save webex @webex/widgets
 ```
 
+## Webex Components vs. Webex Widgets
+
+[Webex Components](https://github.com/webex/components#webex-components) is a set of React components that _only_ follow webex standard styling without any data being injected to them. Versus [Webex Widgets](https://github.com/webex/widgets#webex-widgets) which include the components _plus_ the [sdk adapter](https://github.com/webex/sdk-component-adapter#webex-sdk-component-adapter) to inject the [SDK webex data](https://github.com/webex/webex-js-sdk#webex-js-sdk) accordingly.
```

Comment:
Thanks for writing a first draft on this. I would like to rephrase it some because the way it reads right now it's a little misleading, in the sense that it makes it sound like the components are presentational only. This is not the case.

```suggestion
The [Webex Component System](https://github.com/webex/components#webex-components) is a set of React components that follow Webex styling but for which data is injected via adapters.
This gives you a flexible layout and source of data.

[Webex Widgets](https://github.com/webex/widgets#webex-widgets) are based on [Webex components](https://github.com/webex/components#webex-components) but include the adapter that talks to our [Javascript SDK](https://github.com/webex/webex-js-sdk).
This means that the Webex Widgets use the [SDK Component Adapter](https://github.com/webex/sdk-component-adapter#webex-sdk-component-adapter) to inject the [Webex data](https://github.com/webex/webex-js-sdk#webex-js-sdk) for you.
You also have to take the Widget layout as-is, but the benefit is that there are no configurations needed.
Install, copy-paste and you have the power of Webex in your application!
```